src/library-configuration/ethereum-providers.md

```diff
@@ -0,0 +1,94 @@
+# Ethereum Providers
+
+Resolution libraries require a connection to the Ethereum network to resolve .crypto and .eth domains. To initialize the library, you need to specify an Ethereum node service provider. Once the instance is created you can begin resolving domains. Below are examples of how to initialize the library with different providers.
+
+### Import library
+
+Javascript:
+
+```javascript
+import Resolution from "@unstoppabledomains/resolution";
+```
+
+Java:
+
+```java
+import com.unstoppabledomains.resolution.Resolution
+```
+
+Swift:
+
+```swift
+import UnstoppableDomainsResolution
+```
+
+### Infura
+
+[Infura](https://infura.io/) is a popular Ethereum node service. Obtaining an API key is free and only requires creating an account.
```

Comment:
Let's add a link to alternative Ethereum providers as well: https://ethereum.org/en/developers/docs/nodes-and-clients/nodes-as-a-service/